bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -3,6 +3,7 @@
 	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
 
-	<name>Gardena Smart Home Binding</name>
-	<description>This is the binding for Gardena Smart Home.</description>
+	<name>Gardena Smart System Binding</name>
+	<description>This is the binding for Gardena smart system.</description>
+	<author>Gerhard Riegler</author>
```

Comment:
The author tag has been deprecated and removed in all other bindings, please remove it here too.